File: sdkwork-ai-sdk-swift/Sources/API/I18nResourceApi.swift
import Foundation

public class I18nResourceApi {
    private let client: HttpClient
    
    public init(client: HttpClient) {
        self.client = client
    }

    public func getOptions(params: [String: Any]? = nil) async throws -> PlusApiResultListLabelOption? {
        let response = try await client.get(ApiPaths.aiPath("/v1/open/i18n/options"), params: params)
        return response as? PlusApiResultListLabelOption
    }

    public func createOptions(params: [String: Any]? = nil) async throws -> PlusApiResultListLabelOption? {
        let response = try await client.post(ApiPaths.aiPath("/v1/open/i18n/options"), body: nil, params: params)
        return response as? PlusApiResultListLabelOption
    }
}

File: sdkwork-ai-sdk-csharp/Api/I18nResourceApi.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Ai.Http;
using Ai.Models;

namespace Ai.Api
{
    public class I18nResourceApi
    {
        private readonly HttpClient _client;

        public I18nResourceApi(HttpClient client)
        {
            _client = client;
        }

        public async Task<PlusApiResultListLabelOption?> GetOptionsAsync(Dictionary<string, object>? query = null)
        {
            return await _client.GetAsync<PlusApiResultListLabelOption>(ApiPaths.AiPath("/v1/open/i18n/options"), query);
        }

        public async Task<PlusApiResultListLabelOption?> CreateOptionsAsync(Dictionary<string, object>? query = null)
        {
            return await _client.PostAsync<PlusApiResultListLabelOption>(ApiPaths.AiPath("/v1/open/i18n/options"), null, query);
        }
    }
}
